src/patternfly/components/Tabs/docs/tabs-primary.md

```diff
@@ -0,0 +1,12 @@
+### Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-labelledby="tab1"` | `section` | Identifies which tab is associated with the panel. **Required**  |
+| `hidden`                 | `section` | Indicates that a tab panel is not currently active. **Required** |
+
+### Usage
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `id="tab1"` | `.pf-c-tabs__item` | Uniquely identifies the tab. **Required** |
```

Comment:
should this page have the base classes to create a tabs component?

Reply:
Added `.pf-c-tabs`, `.pf-c-tabs__scroll-button`, `.pf-c-tabs__list`, `.pf-c-tabs__item`, and `.pf-c-tabs__button` to the table. Let me know if that looks correct.